lib/styles/modifiers/_layout.scss

```diff
@@ -1,12 +0,0 @@
-.-centeredSpread {
```

Comment:
moved to platform https://github.com/bethinkpl/wnl-platform/pull/4966/commits/6f98ade1390f7c82ff5ec56f66dc09f278b22277